test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/BusinessEntityControllerSpec.scala

```diff
@@ -544,10 +544,10 @@ class BusinessEntityControllerSpec
 
       "redirect to next page" when {
 
-        val effectiveDate = Seq(
-          fixedDate.plusDays(1).getDayOfMonth,
+        val effectiveDate: String = Seq(
+          fixedDate.plusDays(1).getDayOfMonth.toString,
```

Comment:
It didn't really like the fact that there was an implicit toString happening in the background and has made the signature Seq[Any]